package.json

```diff
@@ -5,7 +5,7 @@
     "appium",
     "unity"
   ],
-  "version": "1.3.2",
```

Comment:
in a PR you shouldn't do things like update the version, that is for the maintainer to do when something is published